package.json

```diff
@@ -40,6 +39,7 @@
     "html-entities": "^1.3.1",
     "js-yaml": "^3.13.1",
     "marked": "^1.2.0",
+    "ramda": "^0.27.1",
```

Comment:
Each page's metadata are now more complex with the addition of localized properties. I'm introducing a dependency on one of my favourite libraries, Ramda, to help with data structure transformations and processing. It brings a lot functional programming concepts to JS.